### PR TITLE
initial version

### PR DESCRIPTION
**Pull request by michaelaye**
ok, so now, with this build you don't have to go through the exercise of creating it yourself with `cookiecutter` but can just browse the build files in GitHub, and then we can comment here what we like and don't like.
I think we should finish that first, before I go ahead and submit the src-layout

### CODE REVIEW

**Review by rbeyer — APPROVED** [review]

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+1. Fork the `package_template_build` repo on GitHub.
+2. Clone your fork locally::
+
+    $ git clone git@github.com:your_name_here/package_template_build.git
+
+3. Install your local copy into a virtualenv. Assuming you have virtualenvwrapper installed, this is how you set up your fork for local development::
+
+    $ mkvirtualenv package_template_build
+    $ cd package_template_build/
+    $ python setup.py develop
```
Comment (michaelaye) [inline CONTRIBUTING.rst]: This needs to change to `pip install -e .` actually, as using `pip` is the favored method over python setup.py due to its cleaner uninstall.

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+   Now you can make your changes locally.
+
+5. When you're done making changes, check that your changes pass flake8 and the
+   tests, including testing other Python versions with tox::
+
+    $ flake8 package_template_build tests
+    $ python setup.py test or pytest
+    $ tox
+
```
Comment (michaelaye) [inline CONTRIBUTING.rst:89]: don't think we should force `tox`.

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+-----------------------
+
+Before you submit a pull request, check that it meets these guidelines:
+
+1. The pull request should include tests.
+2. If the pull request adds functionality, the docs should be updated. Put
+   your new functionality into a function with a docstring, and add the
+   feature to the list in README.rst.
+3. The pull request should work for Python 3.5, 3.6, 3.7 and 3.8, and for PyPy. Check
+   https://travis-ci.org/somedev/package_template_build/pull_requests
```
Comment (michaelaye) [inline CONTRIBUTING.rst:110]: need to update these. we only required 3.6 and 3.7, what about 3.8? Don't required pypy, right?

File: Code-Of-Conduct.rst [hunk 4]
```diff
@@ -0,0 +1,8 @@
+====================================================================
+package_template_build Contributor Covenant Code of Conduct
+====================================================================
+
+The PlanetaryPy Project Code of Conduct][Conduct] applies to
+package_template_build.
```
Comment (michaelaye) [inline Code-Of-Conduct.rst:6]: something wrong with Markdown links here

File: CONTRIBUTING.rst (hunk 1 above)
Comment (rbeyer) [reply CONTRIBUTING.rst]: Fair enough.

File: CONTRIBUTING.rst (hunk 2 above)
Comment (rbeyer) [reply CONTRIBUTING.rst:89]: We aren't forcing anything with this template.  The idea here is to provide a best practice, people are free to do something else.

File: CONTRIBUTING.rst (hunk 3 above)
Comment (rbeyer) [reply CONTRIBUTING.rst:110]: This test is straight from cookiecutter, so yeah, I'd drop the 3.5.  I would consider pypy optional.

File: CONTRIBUTING.rst (hunk 2 above)
Comment (michaelaye) [reply CONTRIBUTING.rst:89]: ah true. i'm thinking too meta (or inverse meta? ;) )

File: Code-Of-Conduct.rst (hunk 4 above)
Comment (rbeyer) [reply Code-Of-Conduct.rst:6]: Yup.

**Comment by michaelaye** [comment]
The Code-of_Conduct page isn't currently listed in a docs build. should i just add it to the main contents page?

**Comment by michaelaye** [comment]
ah sorry, it's outside the docs folder, so it's alright.

File: Code-Of-Conduct.rst (hunk 4 above)
Comment (rbeyer) [reply Code-Of-Conduct.rst:6]: I'd make the link text "Code of Conduct" at the very least.

Comment (michaelaye) [reply Code-Of-Conduct.rst:6]: done

**Pull request merged**